### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by hemantso**
- implemented app component
- implemented booklist, book, book form component
- created reducer and action
- reducer store initialized and connected with the app

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BooksList from '../containers/BooksList';
+import BooksForm from '../containers/BooksForm';
+
+function App() {
```
Comment (wathigo) [inline src/components/App.js]: Let's  make use of es6 syntax like this:
```js
const App = () => { }
```

**Review by wathigo — CHANGES_REQUESTED** [review]
## Changes Required :o:  
Great job on this milestone.

Let's make a few changes before we proceed to the next milestone.
### REQUIRED
**src/reducers/index.js**
* It combines app-specific reducers. The combined reducers should then be passed as the first argument in `createStore()` method in `src/index.js`.

**After making the changes, submit another request for review.**

**Happy Hacking :technologist: 
Simon Wathigo.**

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @hemant-soni-vst-au4 :hand: 
Great job on this milestone so far. 
But we need to make some changes before moving on.

- [x] We need to add a descriptive README file to your repository. The current README file it's the one generated by `create-react-app`. The title is from the Bookstore but it does not have any other description about the project. Please check the instructions [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/), you can use the template to make this process easier.
Screenshot from 2021-01-15 10-47-51

Feel free to message me on Slack if you have any questions.
Happy coding :computer: 
**Sergio Torres**

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi there :wave: 

Go on with the next milestone.

[**note**] Some of your commit messages are not descriptive. Read through the [commit messages](https://github.com/hemant-soni-vst-au4/bookstore/tree/milestone-2) from this link there were spelling errors. Endeavor to write more descriptive commit messages.

### Have a great day :slightly_smiling_face: 

#### ``` Uchechukwu Inyama Henry ```

**Pull request merged**